components/CheckDocumentation/queries/AdvisorInsight.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+
+import { CheckDocs, CheckGuidanceProps } from "../../../util/checks";
+
+const AdvisorInsightGuidance: React.FunctionComponent<
+  CheckGuidanceProps
+> = () => {
+  return (
+    <div>
+      <p>
+        Some common patterns that lead to suboptimal performance are found for
+        this query. You can check out the exact pattern (insight) in the
+        insights section of "Query Summary" panel. Tweaking this query may
+        improve the performance of this query.
+      </p>
+      <h4>Next Step</h4>
+      <p>
+        You can tune a query using Workbooks.
+        <ul>
+          <li>
+            <h5>Create a new workbook and record the baseline</h5>
+            <p>
+              For each insight, there is a "Create Workbook" button to create a
+              new workbook using this query. Create a new workbook using this
+              button, then follow the steps to record the baseline. This is also
+              important step to make sure that the insight can be found with the
+              baseline too. If the query has multiple query plans, it's
```

Comment:
```suggestion
              ```suggestion
              button, then follow the steps to record the baseline. This step
              re-confirms that the insight still reproduces in the baseline too.
              If the query has multiple query plans, it's
```

Reply:
> This step re-confirms that the insight still reproduces in the baseline too.

I think this is grammatically a bit incorrect, should be "still can be reproduced" if we wanna go this way.